src/networks.ts

```diff
@@ -20,3 +20,37 @@ export const CHAIN_IDs = {
   BASE_SEPOLIA: 84532,
   ARBITRUM_SEPOLIA: 421614,
 };
+
+interface PublicNetwork {
+  name: string;
+  nativeToken: string;
+  blockExplorer: string;
+}
+
+export const PUBLIC_NETWORKS: { [chainId: number]: PublicNetwork } = {
```

Comment:
Can we use the CHAIN_IDs to reference? i.e. 
```
    [CHAIN_IDs.MAINNET]: { name: "Mainnet", ... }
```

Reply:
I was concerned about line length with that and feared that it would introduce a lot of line wrapping and make it all look really ugly...but so far only the testnet definitions are long enough to wrap. Implemented: https://github.com/across-protocol/constants-v2/pull/20/commits/dedaef94e8421a4147acfcabc5c4a1fed5da0e83